runtime/cudaq/platform/default/rest/helpers/oqc/oqc.config

```diff
@@ -30,3 +30,22 @@ LIBRARY_MODE=false
 #   --<target>-<option> <value>
 # as in
 #   --oqc-machine foo
+PLATFORM_EXTRA_ARGS=""
+# NB: extra arguments always take the form:
+#   --<target>-<option> <value>
+# as in
+#   --oqc-machine foo
+while [ $# -gt 1 ]; do
+	case "$1" in
+	--oqc-email)
+		PLATFORM_EXTRA_ARGS="$PLATFORM_EXTRA_ARGS;email;$2"
+		;;
+	--oqc-password)
+		PLATFORM_EXTRA_ARGS="$PLATFORM_EXTRA_ARGS;password;$2"
```

Comment:
If this option is used, it will inject the user's password into the executable during the compilation, and I'm concerned that they won't realize that. If they were to share their executable with someone, they would effectively be sharing their password.

Reply:
Agreed. I'll make sure that's env variable only.